docs/examples/cloud_trace_exporter/README.rst

```diff
@@ -24,6 +25,7 @@ To use this exporter you first need to:
 
 .. code-block:: sh
 
+    git clone https://github.com/open-telemetry/opentelemetry-python
     cd opentelemetry-python/docs/examples/cloud_trace_exporter
     python basic_trace.py
```

Comment:
This will run the script from the repo but use the libraries installed from pypi. If the instructions are going to include cloning the repo they may as well include doing an editable install of the packages needed to run the example.

Reply:
Yeah the goal is to install pypi packages and run the script from repo. Shawn just pointed out that it wasn't obvious that the repo needed to be cloned in the first place. I'd like to avoid giving users an editable install of the packages, cuz then they can change stuff and it'll be difficult to help them.